arviz/stats/stats.py

```diff
@@ -464,7 +464,7 @@ def psislw(log_weights, reff=1.0):
         xcutoff = max(x[x_sort_ind[cutoff_ind]], cutoffmin)
 
         expxcutoff = np.exp(xcutoff)
-        tailinds, = np.where(x > xcutoff)
+        tailinds, = np.where(x > xcutoff)  # pylint: disable=unbalanced-tuple-unpacking
```

Comment:
@ahartikainen, is this correct now? Just want to double check

Reply:
Yes, it will return tuple with one item.

Reply:
Great merging